src/main/java/server/ProxyServer.java

```diff
@@ -65,15 +116,18 @@ public TokenIO initializeSdk() throws IOException {
      * @return newly-created member
      */
     private Member createMember(TokenIO tokenIO) {
```

Comment:
We talked about creating a member when a command line param is passed.

Reply:
I decided not to, to keep it simple and get something out. I think it is fine, it allows specifying key directory and alias in the config file. It will try to log in with existing member matching the alias, and if it doesn't exist, it will create a new one